staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -59,6 +59,13 @@ func init() {
 	fatalOnDecodeError, _ = strconv.ParseBool(os.Getenv("KUBE_PANIC_WATCH_DECODE_ERROR"))
 }
 
+// TODO: Added for integration testing of conversion webhooks, where decode errors due to conversion webhook failures need to be tested.
+// Maybe we should identify conversion webhook related errors in decoding to avoid triggering this? Or maybe having this special casing
+// of test cases in production code should be removed?
+func SetFatalOnDecodeError(b bool) {
+	fatalOnDecodeError = b
+}
+
```

Comment:
I need to find a better way to disable this test flag. Suggestions welcome.

Reply:
ugly, but I don't feel too strong about it.

Reply:
I believe to remember that our CI has some mechanism to detect debug symbols in built binaries. I haven't found it on a quick search. Trying with https://github.com/kubernetes/kubernetes/pull/77917. Maybe it was also in OpenShift though. That way one could verify that we do not ship binaries using this hook.

Reply:
Found it: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-symbols.sh

Can we add a pattern or make this func match an existing pattern?

Reply:
Good find. I'll look into it.

Reply:
Looks like `verify-symbols` is no longer doing it's job. Both locally and from a recent `pull-kubernetes-verify` runs I see:

```
Verifying verify-symbols.sh

+++ Running case: verify.symbols 
+++ working dir: /home/prow/go/src/k8s.io/kubernetes
+++ command: bash "hack/make-rules/../../hack/verify-symbols.sh"
+++ [0516 15:44:46] Building go targets for linux/amd64:
    cmd/hyperkube
nm: /home/prow/go/src/k8s.io/kubernetes/_output/local/bin/linux/amd64/hyperkube: no symbols
```

I'll open a separate issue since that hypercube build done by the script has a cost associated with it, and we should only do it if we're going to get some benefit, and without any symbol tables in the binary, we're currently not.

Reply:
https://github.com/kubernetes/kubernetes/issues/77983

Reply:
Fixed: https://github.com/kubernetes/kubernetes/pull/78018

Reply:
Thanks @sttts , I've added the symbol.